### DISCUSSION TITLE
Need help, Database not responding..."Your disk IO budget has run out for today"

### DISCUSSION THREAD

**Discussion by stleeusis** [supabase/supabase#16747] [General]
Hi, my database is currently not responding.  Upon further investigation, it shows "Your disk IO budget has run out for today" and that the Disk IO consumed for the past 2 days is at 100%.  There were no drastic changes that were done prior to this happening or know what could be possibly causing this.
It looks like if I do a report for the last 24 hours the consumption is linear and about 5% growth per hour starting at 4:30 pm yesterday and until it reaches 100% at 9:30 am today.
I already contacted support but got a reply saying "As Pay As You Go plan, we will aim to respond to you in 24-48 hours."
Has anyone encountered this or have insights into this matter?
Thank you
![disk_IO](https://github.com/supabase/supabase/assets/130498219/37f59878-f9f8-44b3-b1e6-0e40fbdd3e96)

**Comment by 260703** [comment]
Yeah, man, it sucks, while facing this issue, which I'm currently facing,
And this supabse support is such a Pu**y boii can't even help, they say upgrade your compute power, and I mean why, I'm currently in the development phase with no users.

BTW, if you solved this problem, please let me know how you solved it.
 :)

**Comment by evelant** [comment]
This appears to be some sort of outage or technical issue not being reported on the supabase status page. We're facing the same thing. Our instance is going totally offline every few hours. Users are furious. Over 24h and no response from support. We're on a paid pro plan. https://github.com/orgs/supabase/discussions/37963

Out of curiosity did you upgrade to the latest postgres version 17.4.1.074 recently? That's the only change we made after which this problem started happening.

**Reply by GaryAustin1** [reply]
@260703 Did you also upgrade versions?
Did both of you get the auto response email with a ticket # at least so you know the contact got thru and you are getting emails?

**Reply by 260703** [reply]
Yes we upgraded from nano to micro, but most likely we have figured out the problem, not quite sure.
as the problem exists in the query optimization, whenever I perform any functionality on my website by clicking a button, and when I inspect the network from developer tools, dauymmm bro so many requests is being sent in bulk, and eventually my supabse got frozen and unresponsive. 

and now I'm banging my head on how to solve the optimization. ;(
